book/src/user_manual/create_key_package.md

```diff
@@ -22,3 +31,6 @@ After creating the key package bundle, clients should store it in the key store
 ```rust,no_run,noplayground
 {{#include ../../../openmls/tests/book_code.rs:store_key_package_bundle}}
 ```
+
+All functions and structs related to key package can be found in the [`key_packages`](https://openmls.tech/openmls/doc/openmls/key_packages/index.html) module.
```

Comment:
Lets link to the official documentation, not main. This will be in both places but I care more about the published version.

Reply:
Right, that was my original intent, but I seem to have confused the two again. Thanks for pointing this out!